README.md

```diff
@@ -37,7 +38,8 @@ const rules = {
   username: 'required|unique:users,username',
   email: 'required|email',
   age: 'required|numeric|min:18',
-  gender: 'in:male,female'
+  gender: 'in:male,female',
+  phoneNumber: 'regex:[0-9]{3} ?-?[0-9]{3} ?-?[0-9]{3}'
```

Comment:
But you can use `,` in regex for example to match date `10th January, 2015`. If you use `,` in regex then you might end up with 2 parameters